code/game/objects/items/weapons/weaponry.dm

```diff
@@ -318,9 +320,14 @@ obj/item/weapon/banhammer/admin
 	projectile.launch_at(A,tar_zone = targeting,from = user)
 
 /obj/item/weapon/bullwhip/attack(mob/M, mob/user)
-	sharpness = 1.2	//a whip can only cut things when it is actually whipping
+	sharpness = 1.2 //a whip can only cut things when it is actually whipping
+	if(user.is_wearing_item(/obj/item/clothing/head/energy_dome))
+		force = force +  whipitgood_bonus
+		if(prob(20))
+			playsound(src,pick('sound/misc/devo_crack_that_whip.ogg','sound/misc/devo_whip_it_good.ogg','sound/misc/devo_you_must_whip_it.ogg'),30)
 	..()
 	sharpness = 0
+	force = force_original//return force back to normal
```

Comment:
force = initial(force)

Reply:
Don't do that
Fuck `initial()`

Reply:
@Exxion What's wrong with initial()?

Reply:
1. Ruins any possibility for adminbus or any kind of other customization
2. If a child of this object were to do something similar to what's happening here it could potentially result in weird behavior if it supercalls (which it would have to in this case)
3. Not relevant in this case but it doesn't work with lists, datums, etc.